### PR TITLE
Remove apps' unnecessary bridge class functionalBlockUtils

### PR DESCRIPTION
**Pull request by bcjordan**
Remove apps' unnecessary bridge class functionalBlockUtils

### CODE REVIEW

File: apps/src/calc/blocks.js [hunk 1]
```diff
@@ -49,35 +46,11 @@ exports.install = function(blockly, blockInstallOptions) {
...
 
-function initFunctionalBlock(block, title, numArgs) {
-  block.setHSV(184, 1.00, 0.74);
-  block.setFunctional(true, {
-    headerHeight: 30,
-  });
-
-  var options = {
-    fixedSize: { height: 35 },
-    fontSize: 25 // in pixels
```
Comment (Bjvanminnen) [inline apps/src/calc/blocks.js]: Probably good to get this using the same code as the blockly-core version, but I'm not sure I'm convinced that they're exactly the same right now. For example, I dont think we get fontSize: 25 using the blockly-core version

**Comment by Bjvanminnen** [comment]
lgtm other than my one comment

File: apps/src/calc/blocks.js (hunk 1 above)
Comment (bcjordan) [inline apps/src/calc/blocks.js]: This function actually appears to have been unused originally, note the reference below is to init**Titled**FunctionalBlock

Comment (Bjvanminnen) [inline apps/src/calc/blocks.js]: Haha, interesting. Cut away!

**Pull request merged**